data/json/bionics.json

```diff
@@ -650,25 +645,26 @@
     "id": "bio_lighter",
     "type": "bionic",
     "name": { "str": "Finger Lighter" },
-    "description": "The index fingers of both hands have powerful fire starters which extend from the tip.",
-    "occupied_bodyparts": [ [ "hand_l", 1 ], [ "hand_r", 1 ] ],
+    "description": "The index finger of your left hand has fire starter which extend from the tip.",
+    "occupied_bodyparts": [ [ "hand_l", 1 ] ],
     "fake_item": "fake_firestarter",
-    "act_cost": "3 kJ"
+    "act_cost": "1500 J"
   },
   {
     "id": "bio_lockpick",
     "type": "bionic",
     "name": { "str": "Fingerpick" },
     "description": "One of your fingers has an electronic lockpick surgically embedded in it.  This automatic system will quickly unlock all but the most advanced key locks without any skill required on the part of the user.",
-    "occupied_bodyparts": [ [ "hand_r", 2 ] ],
+    "occupied_bodyparts": [ [ "hand_l", 2 ] ],
```

Comment:
you didn't even edit the slots here, looks a bit pointless to just shift it to the other side.

Reply:
Similar logic to the above, I suspect.